Source/reactions/Castro_react_util.H

```diff
@@ -0,0 +1,265 @@
+#ifndef CASTRO_REACT_UTIL_H
+#define CASTRO_REACT_UTIL_H
+
+#include <Castro.H>
+#include <burn_type.H>
+// #include <actual_rhs.H>
+
+AMREX_GPU_HOST_DEVICE AMREX_INLINE
+void
+single_zone_react_source(GpuArray<Real, NUM_STATE> const& state,
+                         GpuArray<Real, NUM_STATE>& R,
+                         burn_t burn_state);
+
```

Comment:
why isn't this implemented?

Reply:
oh... I think you don't need this prototype.  You just need to reorder the functions in dependency order in the header.